packages/dds/merge-tree/src/test/testClientLogger.ts

```diff
@@ -79,6 +79,15 @@ export class TestClientLogger {
     // initialize to private instance, so first real edit will create a new line
     private lastDeltaArgs: IMergeTreeDeltaOpArgs | undefined;
 
+    private readonly disposeCallbacks: (() => void)[] = [];
+
+    public dispose(): void {
```

Comment:
IMO worth adding a small doc comment

```
/**
 * Unsubscribes this logger from its clients' events. Consider using this for tests with client lifetime
 * extending significantly past the logger's.
 */
```

Shouldn't cause any correctness issues to not call it for smaller tests, so I think that's sufficient.

@anthony-murphy FYI once you're back in. Added this to resolve memory leak issues in fuzz tests (they'd construct lots of loggers which would all remain subscribed to all of the clients involved).

Reply:
Added

Reply:
should be above `dispose`, not the private field :)